launcher/ui/pages/modplatform/ResourceModel.cpp

```diff
@@ -77,7 +77,7 @@ auto ResourceModel::data(const QModelIndex& index, int role) const -> QVariant
         case UserDataTypes::TITLE:
             return pack->name;
         case UserDataTypes::DESCRIPTION:
-            return pack->description;
+            return pack->description.simplified();
```

Comment:
I would prefer that the simplified call be made in ProjectItem.cpp on "Description painting" part removing the need to modify the ResourceModel and  ModrinthModel

Reply:
understandable, pushing a fix now